libr/core/cmd_anal.c

```diff
@@ -4116,21 +4116,21 @@ static bool anal_fcn_data (RCore *core, const char *input) {
 		int i;
 		bool gap = false;
 		ut64 gap_addr = UT64_MAX;
-		char *bitmap = calloc (1, fcn->size);
+		char *bitmap = calloc (1, r_anal_fcn_size (fcn));
 		if (bitmap) {
 			RAnalBlock *b;
 			RListIter *iter;
 			r_list_foreach (fcn->bbs, iter, b) {
 				int f = b->addr - fcn->addr;
-				int t = R_MIN (f + b->size, fcn->size);
+				int t = R_MIN (f + b->size, r_anal_fcn_size (fcn));
```

Comment:
fcn size will not change in the middle of the loop. so calling this function all the time is overkill